client/src/pages/CreateAction.js

```diff
@@ -1,25 +1,154 @@
 import "./CreateAction.css";
 import Card from "../components/Card";
-import HeaderGoForward from "../components/Card_components/HeaderGoForward";
+import HeaderActionGoForward from "../components/Card_components/HeaderActionGoForward";
 import SpeechInput from "../components/SpeechInput";
-import { useParams } from "react-router";
+import { useParams, useHistory } from "react-router";
+import { useState, useEffect } from "react";
 import FooterSubmit from "../components/Card_components/FooterSubmit";
 
 export default function CreateCase() {
+  /** VARIABLE SET UP */
+  const reminderArray = JSON.parse(localStorage.getItem("newEntry"));
   const { number } = useParams();
+  const indexNumber = Number(number);
+  const history = useHistory();
+  const [verbInput, setVerbInput] = useState(
+    reminderArray.tasks[indexNumber - 1].verb
+  );
+  const [actionInput, setActionInput] = useState(
+    reminderArray.tasks[indexNumber - 1].action
+  );
+  const [withInput, setWithInput] = useState(
+    reminderArray.tasks[indexNumber - 1].with
+  );
+  const [isTooShort, setIsTooShort] = useState(true);
+
+  /** CHECK FOR INPUT */
+  useEffect(() => {
+    verbInput.length === 0 ? setIsTooShort(true) : setIsTooShort(false);
+    actionInput.length === 0 ? setIsTooShort(true) : setIsTooShort(false);
+  }, [verbInput, actionInput, isTooShort]);
+
+  /** ONCLICK ADD NEW TASK */
+  function caseAddGoForward(event) {
+    event.preventDefault();
+
+    if (isTooShort) {
+      return;
+    }
+
+    const taskObj = {
+      taskId: indexNumber,
+      verb: verbInput,
+      action: actionInput,
+      with: withInput,
+    };
+
+    reminderArray.tasks[indexNumber - 1] = taskObj;
+    reminderArray.tasks.push({
+      taskId: indexNumber + 1,
+      verb: "",
+      action: "",
+      with: "",
+    });
+
+    localStorage.removeItem("newEntry");
+    localStorage.setItem("newEntry", JSON.stringify(reminderArray));
```

Comment:
I have the feeling here  you don't want add the reminderArray to `newEntry` but to `reminder`? 

Somehow in my local storage i only have this `reminder` key that is an object, and I never get an array:

![image](https://user-images.githubusercontent.com/6980777/130644049-f0349865-76f2-4ddb-a0ed-f542e1dedd1c.png)